packages/api/src/objectSet/ObjectSetListener.ts

```diff
@@ -55,3 +55,20 @@ type ObjectUpdate<
   object: Osdk.Instance<O, never, P>;
   state: "ADDED_OR_UPDATED" | "REMOVED";
 };
+
+/**
+ * Options for subscribing to an ObjectSet.
+ *
+ * @property properties - The properties to request a subscription for. Requesting specific properties limits the possible properties
+ * that can be returned from the subscription. If not provided, all properties will be requested and potentially be returned on updates.
+ *
+ * @property requestLatestGeotimeSeriesValues - If true, any geotime series properties will be updated with the latest values. This can
+ * reduce performance and should be used only if needed.
+ */
+export interface ObjectSetListenerOptions<
```

Comment:
I want to export this from the API similar to other Options types we have, is there anything else to keep in mind?

Reply:
Seems reasonable to export from the main index.ts